ethereum/stratum/src/main/java/org/hyperledger/besu/ethereum/stratum/Stratum1EthProxyProtocol.java

```diff
@@ -93,9 +94,9 @@ public void onClose(final StratumConnection conn) {}
   public void handle(final StratumConnection conn, final String message) {
     try {
       JsonRpcRequest req = new JsonObject(message).mapTo(JsonRpcRequest.class);
-      if ("eth_getWork".equals(req.getMethod())) {
-        sendNewWork(conn, req.getId());
-      } else if ("eth_submitWork".equals(req.getMethod())) {
+      if ("eth_getWork".equals(req.getRequestBody().getMethod())) {
```

Comment:
nit: extract req.getRequestBody().getMethod() to local var

Reply:
think there's something wrong here - have changed this to use a JsonRequestBody ... the Json Deserialise was never going to work ... :/